jupyterlab_wrattler/src/index.ts

```diff
@@ -14,7 +14,17 @@ const CSS_CLASS = 'jp-Wrattler';
  */
 export const MIME_TYPE = 'text/plain';
 /** @hidden */
-export const USE_BINDER= false
+export const USE_BINDER= getUseBinder();
+
+function getUseBinder() {
+  let baseURL:string = window.location.hostname
+  console.log("BaseURL is: ".concat(baseURL))
+  if ((baseURL == "127.0.0.1") || ('localhost'))
```

Comment:
Should this be ```if ((baseURL == "127.0.0.1") || (baseURL == 'localhost'))``` ?

Also, I think maybe we should also look for "0.0.0.0", i.e. 
```if ((baseURL == "127.0.0.1") || (baseURL == 'localhost') || (baseURL == "0.0.0.0"))```

Reply:
Fixed here: https://github.com/wrattler/wrattler/blob/8d448a06700bc84e217122ceb8543ca7a7747468/jupyterlab_wrattler/src/index.ts#L22